pages/apis/rest_api/builds.md

```diff
@@ -109,9 +109,16 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pipeline.
         "started_at": "2015-05-09T21:07:59.874Z",
         "finished_at": "2015-05-09T21:08:59.874Z",
         "retried": false,
-	"retried_in_job_id": null,
-	"retries_count": null,
-	"retry_type": null
+        "retried_in_job_id": null,
+        "retries_count": null,
+        "retry_type": null,
+        "parallel_group_index": null,
+        "parallel_group_total": null,
+        "matrix": null,
+        "cluster_id": null,
+        "cluster_url": null,
+        "cluster_queue_id": null,
+        "cluster_queue_url": null
```

Comment:
Are the any meaningful values we could use here instead of `null`? We're not the best at including example values for fields so these responses do a lot of heavy lifting for us.

Reply:
I think we could definitely look to expand on some of these in the future, but in this case I want to get this (long overdue!) API doc update out, so I'm merging this now (after applying all of your other feedback, thank you!)